tests/api/test_product_minimal_variant_price.py

```diff
@@ -295,9 +295,12 @@ def test_category_delete_updates_minimal_variant_price(
     data = content["data"]["categoryDelete"]
     assert data["errors"] == []
 
-    mock_update_products_minimal_variant_prices_task.delay.assert_called_once_with(
-        product_ids=[p.pk for p in product_list]
-    )
+    mock_update_products_minimal_variant_prices_task.delay.assert_called_once()
+    (
+        call_args,
```

Comment:
`call_args` could be `_`

Reply:
True, I wish python had better destructuring of dictionaries.